reflex/utils/processes.py

```diff
@@ -282,3 +283,40 @@ def show_progress(message: str, process: subprocess.Popen, checkpoints: List[str
 def atexit_handler():
     """Display a custom message with the current time when exiting an app."""
     console.log("Reflex app stopped.")
+
+
+def run_process_with_fallback(args, *, show_status_message, fallback=None, **kwargs):
+    """Run subprocess and retry using fallback command if initial command fails.
+
+    Args:
+        args: A string, or a sequence of program arguments.
+        show_status_message: The status message to be displayed in the console.
+        fallback: The fallback command to run.
+        kwargs: Kwargs to pass to new_process function.
+    """
+
+    def execute_process(process):
+        if not constants.IS_WINDOWS:
+            show_status(show_status_message, process)
+        else:
+            process.wait()
```

Comment:
Not a fan of this but this was a way to get it to work. Here's the use case for this:
`subprocess.Popen` is non-blocking which is what we use conventionally, `show_status` prints the output of the spawned process. 
Using `show_status` in this case means errors will still be displayed when the process throws one in INFO mode (I find it intuitive to have this error in debug mode instead).
In the case of running with a fallback, we want to know whether the process failed or not, so we wait(which is blocking) for completion of the process. I tested this against reflex-web and the perf difference (between the case of waiting and not waiting) was not noticeable. Open to suggestions, though.